### DISCUSSION TITLE
How to use android queue mode

### DISCUSSION THREAD

**Discussion by 4-alok** [dlutton/flutter_tts#213] [Q&A]
## 💬 Questions and Help

Please give an example, how does **setQueueMode(queueMode)** work

**Comment by dlutton** [comment]
@4-alok you'd set it to 1 or 0.


https://developer.android.com/reference/android/speech/tts/TextToSpeech#QUEUE_ADD

```
Queue mode where the new entry is added at the end of the playback queue.

Constant Value: 1 (0x00000001)
```

https://developer.android.com/reference/android/speech/tts/TextToSpeech#QUEUE_FLUSH

```
Queue mode where all entries in the playback queue (media to be played and text to be synthesized) are dropped and replaced by the new entry. Queues are flushed with respect to a given calling app. Entries in the queue from other callees are not discarded.

Constant Value: 0 (0x00000000)
```

**Reply by nt4f04uNd** [reply]
why is it not bool?

**Reply by nt4f04uNd** [reply]
(i mean, in your package)

**Reply by dlutton** [reply]
@nt4f04uNd I could also set this as a bool.  I'm not sure how other developers use these methods, so feedback and PRs are always welcome.
